src/viv-imx6/view-backend.cpp

```diff
@@ -42,6 +42,9 @@ struct ViewBackend : public IPC::Host::Handler, public WPE::LibinputServer::Clie
     ViewBackend(struct wpe_view_backend*);
     virtual ~ViewBackend();
 
+    uint32_t width { WIDTH };
+    uint32_t height { HEIGHT };
+
```

Comment:
This change looks not needed? why moving this member variables above?

Reply:
not really. Already reverted in the latest revision of the PR.